packages/components/src/entities/SamplesTabbedGridPanel.tsx

```diff
@@ -372,8 +375,7 @@ export const SamplesTabbedGridPanel: FC<Props> = memo(props => {
                     labelTemplate={labelTemplate}
                     printServiceUrl={printServiceUrl}
                     onCancel={onCancelPrint}
-                    queryName={activeModel?.queryName}
-                    schemaName={activeModel?.schemaName}
+                    model={printDialogModel} // TODO change more usages to this
```

Comment:
Let's do this now unless there's a reason it's complicated. If other usages don't have a QueryModal in hand, perhaps the properties for this component should instead be changed to just take the optional filters to be used. I think that will simplify `PrintLabelsModal` a bit.

Reply:
done